docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+
+  database:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_PASSWORD=development
+
+  app:
+    image: github-inbox
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - 3000:3000
```

Comment:
I think you're right.  If 1.9 was a few months older, it would be worth doing.  Maybe in April...